pkg/build/controller/strategy/custom.go

```diff
@@ -22,12 +23,22 @@ type CustomBuildStrategy struct {
 
 // CreateBuildPod creates the pod to be used for the Custom build
 func (bs *CustomBuildStrategy) CreateBuildPod(build *buildapi.Build) (*kapi.Pod, error) {
-	data, err := bs.Codec.Encode(build)
```

Comment:
Yes, all strategies have their codecs set in run_components.go:
https://github.com/openshift/origin/blob/master/pkg/cmd/server/origin/run_components.go#L190-L191
https://github.com/openshift/origin/blob/master/pkg/cmd/server/origin/run_components.go#L203-L218
Should we just use latest?